### ISSUE TITLE
404 Not Found: index.html

### ISSUE THREAD

**Issue by mzohaibnaz** [Samuel-Martineau/generator-svelte#6]
**Describe the bug**
i just created a project with command **"yo svelte"** with all first default options just hitting enter,
after installation done, i tried to run it on browser with command "**npm run dev**" but something is wrong,

**Logs**
```
404 Not Found

D:\NodeJS\todo\public\index.html

(rollup-plugin-serve)

```

**Issue on build**
and even i tried to build that project with command "**npm run build**"
i'm getting some kinda errors ,

*Logs*
```
[!] (plugin at position 7) Error: ENOENT: no such file or directory, open 'D:\NodeJS\todo\public\index.html'
Error: ENOENT: no such file or directory, open 'D:\NodeJS\todo\public\index.html'
    at Object.fs.openSync (fs.js:579:3)
    at Object.fs.writeFileSync (fs.js:1309:33)
    at Object.generateBundle (D:\NodeJS\todo\rollup.config.js:30:8)
    at Promise.resolve.then (D:\NodeJS\todo\node_modules\rollup\dist\shared\rollup.js:18269:25)
```

**Comment by Samuel-Martineau** [comment]
Which version of generator-svelte are you using?

**Comment by mzohaibnaz** [comment]
well,

i just installed with the given command "**npm install -g yo generator-svelte**".
and it should install the latest one for me ... because i'm not specifying any version.

**Comment by Samuel-Martineau** [comment]
I'm looking into this, but I really don't understand what is causing this. Our [unit tests](https://github.com/Samuel-Martineau/generator-svelte/actions/runs/178802269) work on Windows, so the problem isn't with the OS...

**Comment by Samuel-Martineau** [comment]
Could you give me the outputs of these commands on your computer:
- **npm list -g --depth=0**
- **node -v**
And did you changed anything between generating the app and trying to run it?

**Comment by Samuel-Martineau** [comment]
I added more unit tests and it seems like this generator doesn't work on windows. I'm looking into it.

**Comment by Samuel-Martineau** [comment]
This seems to be linked with #3 , which I thought was fixed

**Comment by Samuel-Martineau** [comment]
Fixed in 58c2311e9335016494dd972c9949cad581ab1eab
